package/whiteboard_hypercard/package

```diff
@@ -0,0 +1,30 @@
+# vim: set ft=sh:
+pkgname=whiteboard_hypercard
+pkgdesc="Real-time collaboration, drawing or whiteboarding"
+url=https://github.com/fenollp/reMarkable-tools
+pkgver=0.3.0
+timestamp=2020-09-21T17:59Z
+section=games
+maintainer="Pierre Fenoll <pierrefenoll@gmail.com>"
+license=Apache-2.0
+
+image=rust:v1.1
+source=(https://github.com/fenollp/reMarkable-tools/archive/v0.3.0.zip)
+sha256sums=(3d160d77cd384ee9339c7f791b3f762bb295a670c0c728373c6a09f14a061046)
+
+build() {
+    cd marauder
+    rustup component add rustfmt --toolchain nightly-x86_64-unknown-linux-gnu
+    cargo build --release --bin whiteboard --locked #--frozen #--offline
+    arm-linux-gnueabihf-strip target/armv7-unknown-linux-gnueabihf/release/whiteboard
+}
+
+package() {
+    echo 'name=whiteboard'                >"$srcdir"/whiteboard_hypercard.draft
```

Comment:
I'd suggest moving this to the build section, or echoing directly to the `$pkgdir` file

Reply:
Actually, since I see that you've included the `*.draft` file already, these lines should be removed.

Reply:
Ah yes i forgot about that. 
What's the name of the variable that is the path to the folder containing the draft (and package) file?